webapp2.py

```diff
@@ -72,6 +72,13 @@
                     "so webapp2 won't be thread-safe!")
 
 
+def iteritems(d):
```

Comment:
unless this is intended to become part of the public API of webapp2, please name it `_iteritems`.